test/blackbox-tests/mel-send-pipe-expr.t

```diff
@@ -0,0 +1,33 @@
+Showcase `@mel.send.pipe` with an expression
+
+  $ . ./setup.sh
+  $ cat > x.ml <<EOF
+  > external f : int -> int -> int array -> unit = "f" [@@mel.send.pipe: int]
+  > external g : int -> int -> int array -> unit = "g" [@@mel.send.pipe int]
```

Comment:
My first impression is that this makes things a bit more confusing, because now there are 2 ways to express the same thing. Is there a way to mitigate this? E.g. we could deprecate the old way if we prefer the new one.

Reply:
> E.g. we could deprecate the old way if we prefer the new one.

that's not a valid alternative, given my third point in the PR description:

> A disadvantage of this approach is that you can't really write [@mel.send.pipe 'a array] (in which case you still need the previous syntax).

Expressions don't have type variables.

Reply:
In this case, I think I prefer keeping just the old approach. I am not sure that in order to save a keystroke for `:` it is worth to end up with 2 ways of doing the same thing. 🤔 wdyt?

Reply:
Yeah, I agree. This is starting to sound like a bad idea.